src/browser.js

```diff
@@ -148,35 +204,20 @@ function save(namespaces) {
  */
 
 function load() {
-  try {
-    return exports.storage.debug;
-  } catch(e) {}
-
-  // If debug isn't set in LS, and we're in Electron, try to load $DEBUG
-  if (typeof process !== 'undefined' && 'env' in process) {
-    return process.env.DEBUG;
+  if (exports.storage) {
+    exports.storage.get('debug', exports.enable);
```

Comment:
That is correct, I am indeed pointing that out.

Basically what you are arguing is that this PR is making the situation better, as `debug` goes from not working at all to working in most cases. I can buy that, but I don't buy that the risk is small. I 100% consistently saw that messages that were logged by a logger assigned directly after the `import debug from 'debug';` did not appear.

Let's make it clear that I'm not saying it's a deal breaker. It would be awesome to `debug` working. We just may need to make clear that the following is the way you need to be logging in a WebExtension until we can improve it further. Example using React.js:

```javascript
import debug from 'debug';
// const log = debug('FooComponent'); // <-- DO NOT DO THIS

export default function FooComponent() {
  debug('FooComponent')('Rendering FooComponent');
  // log('Rendering FooComponent'); // <-- DO NOT DO THIS

  return <h1>Foo</h1>;
}
```